Vagrantfile

```diff
@@ -0,0 +1,85 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
```

Comment:
Remove those comments. They are cluttering the code.

Reply:
lines 4-7 are related.
so to delete comments to 4-7 as well?

Reply:
delete all comments. Use comments only to explain the reasoning for doing something in the way it was done.

Reply:
hey i left only the relevent comments
and commented the configuration for the boostrap that will be uncommented later

Reply:
still too many comments

Reply:
hey i updated the file

Reply:
deleted all the comments in the vagrantfile